content/momentum/4/http-api-stats/queues-mailq.md

```diff
@@ -0,0 +1,107 @@
+---
+lastUpdated: "02/01/2023"
+title: "Mail Queue"
+description: "/stats/queues/mailq show the status of the mail queues When issued with no arguments this command shows counts for the active queue and the delayed queues sorted by binding groups and bindings This request shall be done to the port 2081 using any..."
+---
+
+<a name="http_api_stats.queues_mailq"></a>
+## Name
+
+`/stats/queues/mailq` — show the status of the mail queues
+
+## Synopsis
+
+`GET /stats/queues/mailq`
+
+`GET /stats/queues/mailq?domain=some.domain`
+
+`GET /stats/queues/mailq?binding=ip_1.2.3.4`
+
+`GET /stats/queues/mailq?ip=1.2.3.4`
```

Comment:
We should remove this example of `ip` query parameter.  on prem would always be referencing "binding" instead of ip.